docs/src/recommended-configuration.md

```diff
@@ -62,6 +62,31 @@ The biggest blue element will be you `docSearch-content` container. Every
 selectors outside this element will be `global`. Every selectors appear in the
 same order than their `lvl` along the HTML flow.
 
+### Introduces global information as meta tags
+
+Our crawler automatically extract information from our DocSearch specific meta
+taf:
+
+```html
+<meta name="docsearch:language" content="en">
+<meta name="docsearch:version" content="1.0.0">
+```
+
+The `content` value of the meta will be added to every records extracted from
+the page. Giving that the name is `docsearch:$NAME`, `$NAME` will be set as a
+attribute in every of them. Its value will be its related `content`value. You
+can then transform them as [`facetFilter`][6] to let you filter over it.
+
+Look at this example with our vuepress integration:
+
+```json
+"custom_settings": {
+    "attributesForFaceting": [
+      "lang"
+    ]
+  }
+```
+
```

Comment:
You're mentioning `facetFilters` (for the front) above, then here using `attributesForFaceting` (for the settings). I think it needs a bit more clarification.